specs/test_formats/ssz/uint.md

```diff
@@ -0,0 +1,23 @@
+# Test format: SSZ uints
+
+SSZ supports encoding of uints up to 32 bytes. These are considered to be basic types.
+
+## Test case format
+
+```yaml
+type: "uintN"      -- string, where N is one of [8, 16, 32, 64, 128, 256]
+valid: bool        -- expected validity of the input data
+value: string      -- string, decimal encoding, to support up to 256 bit integers
+ssz: bytes         -- string, input data, hex encoded, with prefix 0x
```

Comment:
Yep, remember going over this when documenting the test format. For now I'll just keep it compatible with the existing ssz format. With the future ssz testing additions this can be improved :+1: